### PR TITLE
[VEN-2715] Add weETHs Market

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN

### CODE REVIEW

File: multisig/proposals/sepolia/vip-052/index.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    {
+      target: sepolia.POOL_REGISTRY,
+      signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+      params: [[vweETHs, CF, LT, INITIAL_SUPPLY, sepolia.VTREASURY, SUPPLY_CAP, BORROW_CAP]],
+    },
+    {
+      target: vweETHs,
+      signature: "setProtocolSeizeShare(uint256)",
+      params: [parseUnits("0.01", 18)],
+    },
```
Comment (chechu) [inline multisig/proposals/sepolia/vip-052/index.ts:65]: We have to pause the borrow action, as we made with the Pendle token

**Review by chechu — COMMENTED** [review]
`multisig/proposals/sepolia/vip-052/addendum.ts` lgtm

**Review by chechu — APPROVED** [review]

**Review by GitGuru7 — APPROVED** [review]
LGTM!

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/355?chainId=56

**Pull request merged**